app/controllers/movie_preference_controller.py

```diff
@@ -20,3 +21,17 @@ def handle(self, user_id):
         db.session.commit()
 
         return jsonify(mp.to_dict())
+
+    def get(self, user_id):
+        user = get_current_session_user()
+        if user is not User.query.get(user_id):
+            abort(403)
+
+        movie_ids = [mp.external_movie_id for mp in user.movies]
+
+        movies = []
+        for id in movie_ids:
+            movie = get_movie(id)
```

Comment:
Ya this is where we'll hit rate limiting but w/e for now

Reply:
It also makes the requests synchronously, so the more movies you add the slower it gets :( In a followup I'll look into making them parallel (see http://skipperkongen.dk/2016/09/09/easy-parallel-http-requests-with-python-and-asyncio/), but I wanna do more research on it first before picking a solution